### ISSUE TITLE
Consider sign the assembly?

### ISSUE THREAD

**Issue by AsakusaRinne** [mcintyre321/OneOf#147]
Hi, I'm using `Oneof` in our project and thanks a lot for all your works. For some reason, my project is required to sign the assembly. When using our package with .NET core, there's no problem. However when using .NET Framework (for example, 4.8), the compilation will fail and say `Oneof.dll` is not signed. 

Since the only dependency of `Oneof` is `Microsoft.SourceLink.GitHub`, I think signing `Oneof` assembly will not bring negative effects Could you please consider it?

**Comment by AsakusaRinne** [comment]
Yes, what #38 asked is exactly the same. Sorry that I didn't notice it. I still want to kindly ask if you consider signing the assembly since it won't bring negative effect. shall we move to #38 to talk about it?

**Comment by bartelink** [comment]
Yes, closing is good. But I'm just a lurker; I feel for the maintainers about busywork entering a process, and I feel for you being forced into 27b/4 processes - I hope you can get out of it ;)

**Comment by AsakusaRinne** [comment]
Thank you very much :)
